kartograf/merge.py

```diff
@@ -131,6 +147,26 @@ def extra_file_to_df(extra_file_path):
 
     return df_extra
 
+def process_chunk_worker(chunk_data, base_dict):
+    base = BaseNetworkIndex.from_dict(base_dict)
```

Comment:
On my latest pass I have noticed that we are pickling this per chunk instead of per worker. There should be a way to pickle this per worker and keep it for as many chunks as the worker is processing. But we can keep this as a follow-up optimization, maybe along with the tasks per worker idea. The performance impact would need to be measured across different setups to check if there is any serious impact.